src/debug/di/shimremotedatatarget.cpp

```diff
@@ -215,8 +215,10 @@ ShimRemoteDataTarget::GetPlatform(
          *pPlatform = CORDB_PLATFORM_POSIX_X86;
      #elif defined(DBG_TARGET_AMD64)
          *pPlatform = CORDB_PLATFORM_POSIX_AMD64;
-    #elif defined(DBG_TARGET_ARM)
+     #elif defined(DBG_TARGET_ARM)
```

Comment:
.. the higher decl is at 5 spaces, so that's where things ended up.  What are we using for this?  Farther up the diff you changed everything from 5 to 4.